templates/tfengine/recipes/terragrunt_deployment.hcl

```diff
@@ -12,6 +12,85 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+schema = {
```

Comment:
This schema looks duplicated. Is there no way to un-duplicate?

Reply:
I think in a follow up PR I will merge the two into one recipe called deployment and that will have a flag for enabling terragrunt.